### DISCUSSION TITLE
Welcome to centralized-relay Discussions!

### DISCUSSION THREAD

**Discussion by debendraoli** [icon-project/centralized-relay#128] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DeepakBomjan** [comment]
Hi I am trying to install the centralized-relay, running `make install`, below error is seen:
<img width="1280" alt="image" src="https://github.com/icon-project/centralized-relay/assets/44976635/deffff15-9a80-4a54-aa5f-1e3c972f79da">
